docs/readmes/architecture/agw_overview.md

```diff
@@ -0,0 +1,59 @@
+---
+id: agw_overview
+title: AGW Overview
+hide_title: true
+---
+
+# Access Gateway Overview
+
+The main service within the access gateway (AGW) is Magmad, which brings up all the services hosted within the AGW. The other major services and components hosted within the AGW are: the MME, S-PGW, Health Checker, Mobilityd, Sessiond/PCEF, Pipelined, PolicyDB, Subscriberdb, OVS-data path, Enodebd, and the Control Proxy. Together, these components help to facilitate and manage data both to and from the user. 
+
+![AGW diagram](assets/agw_services.png)
+
+## Access Gateway Services and  Components
+
+1. **Sctpd** - High availability SCTP interface to radio front end. Sctpd forms the endpoint of the S1-C connection. 
+
+2. **MME** - Implements S1AP, NAS and MME subcomponents for LTE control plane Also implements SGW and PGW control plane. If the mme is restarted, S1 connection will be restarted and users service will be affected. Mobilityd, pipelined, and sessiond are restarted in this process as well. 
+
+3. **Enodebd** - Enodebd supports management of eNodeB devices that use TR-069 as a management interface. This is used for both provisioning the eNodeB and collecting the performance metrics. It also acts as a statistics reporter for externally managed eNodeBs. It supports following data models:
+	* Device Data model : TR-181 and TR-098
+	* Information Data model : TR-196
+
+4. **Magmad** - Parent service to start all Magma services, owns the collection and reporting of metrics of services, and also acts as the bootstrapping client with Orchestrator.
```

Comment:
@ssanadhya, @ardzoht, @pshelar  could you double check the first part of the statement, i.e., start all Magma services?

Reply:
Yeah, basically magmad acts as the manager to kick start the systemd services managed by it.